*** timezone.py	2013-06-23 20:42:06.000000000 +0200
--- timezone-new.py	2013-06-23 20:49:41.000000000 +0200
***************
*** 17,23 ****
          return value
  
      def now():
!         return timezone.localtime(timezone.now())
  
  except ImportError:
      now = datetime.datetime.now
--- 17,26 ----
          return value
  
      def now():
! 	d = timezone.now()
! 	if d.tzinfo:
! 	    return timezone.localtime(timezone.now())
! 	return d 
  
  except ImportError:
      now = datetime.datetime.now
